libcxx/include/support/atomic/atomic_base.h

```diff
@@ -0,0 +1,259 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Part of libcu++, the C++ Standard Library for your entire system,
+// under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef _LIBCUDACXX_ATOMIC_BASE_H
+#define _LIBCUDACXX_ATOMIC_BASE_H
+
+#include <type_traits>
```

Comment:
If a file including this file doesn't include type_traits, type_traits is included inside a namespace, right? This needs to stop being here.

Reply:
:+1: Thanks for the catch.